website/contribute/developer-setup.md

```diff
@@ -275,6 +275,20 @@ Discussion about contributing code to Hudi happens on the [dev@ mailing list](/c
 
 This Quick Video will give a code walkthrough to start with [watch](https://www.youtube.com/watch?v=N2eDfU_rQ_U).
 
+## Running unit tests and local debugger via Intellij IDE
+
+#### IMPORTANT REMINDER FOR BELOW STEPS: When submitting a PR please make sure to NOT commit the changes mentioned in these steps, instead once testing is done make sure to revert the changes and then submit a pr.
+
+0. Build the project with the intended profiles via the `mvn` cli, for example for spark 3.2 use `mvn clean package -Dspark3.2 -Dscala-2.12 -DskipTests`. 
+1. Install the "Maven Helper" plugin from the Intellij IDE.
+2. Make sure IDEA uses Maven to build/run tests:
```

Comment:
We should note that the developer should click the checkbox for the "Skip Tests" option so they don't have to wait on the tests to run as part of the intellij build process